app/controllers/api/v1/background_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::BackgroundController < ApplicationController
+
+  def show
```

Comment:
Single image for background, so created a show method within the background controller.